### PR TITLE
Make Contributing Guide more concise

### PR DESCRIPTION
**Pull request by hennevogel**
I've missed the discussion about changing this, so here is my feedback for CONTRIBUTING.md in the form of a PR :smile: 

I think in general this needs to be as concise **and** as clear as possible because this is mostly for new contributors to learn. We should try not to overwhelm then in the sections.

### CODE REVIEW

**Comment by hellcp** [comment]
I do find templates a little easier to follow compared to guidelines for the pr. Maybe something like:

```
### Description
<!--- Explain what this PR is about. -->

### How to see the change
<!--- Explain the steps the reviewer has to follow to verify your change.
(If you need need sample data to verify your change, please explain
how to create that data)

Additionally, if you address performance in this PR, add benchmark data
or explain how the reviewer can benchmark this. -->

<!--- If your change was visual, include before and after screenshots
and remove `<!---`, `\-->` comment tags before and after this block
### Screenshots
**Before**

**After**

-->
```
Filling out forms will always be easier than inventing the wheel on your own. And making it short and sweet is a bonus ;)

**Comment by hennevogel** [comment]
> I do find templates a little easier to follow compared to guidelines for the pr.

Hm, I find it a bit hard then to understand what we would expect to fill in that form...

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,7 +1,45 @@
...
+how we work and what we expect from all contributors.
 
 https://github.com/openSUSE/open-build-service/blob/master/CONTRIBUTING.md
+
+In order to make it as easy as possible for other developers to review your
+pull request we ask you to:
+
+- Explain what this PR is about in the description.
+- Explain the steps the reviewer has to follow to verify your change.
+- If you need need sample data to verify your change, please explain how to
```
Comment (krauselukas) [inline .github/PULL_REQUEST_TEMPLATE.md]: Typo: 2 x "need"

**Comment by krauselukas** [comment]
Besides the typo, it looks good to me :)

**Comment by hennevogel** [comment]
@openSUSE/obs-frontend any other opinion?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,7 +1,45 @@
-
-
 <!---
-If you haven't done so already, please read the CONTRIBUTING.md file to learn how we work and what we expect from all contributors.
+If you haven't done so already, please read the CONTRIBUTING.md file to learn
```
Comment (dmarcoux) [inline .github/PULL_REQUEST_TEMPLATE.md]: I don't really like that this template is now huge. It's noise for any contributor which already contributed.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -142,43 +97,70 @@ with us in the pull-request, over the
 developer [mailing list](https://lists.opensuse.org/obs-devel) or our
 IRC channel (irc://irc.freenode.net/openSUSE-buildservice).
 
-# The Perfect Pull Request Review
+# How to review code submissions
+
+Prerequisites: familiarity with [GitHub pull request reviews](https://help.github.com/articles/about-pull-request-reviews).
+
+We belive every code submission should be reviewed by another developer to determine it's *maintainablity*.
```
Comment (dmarcoux) [inline CONTRIBUTING.md]: beli*e*ve

it*s* (no apostrophe, it's a pronoun)

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (hennevogel) [reply .github/PULL_REQUEST_TEMPLATE.md]: > I don't really like that this template is now huge. It's noise for any contributor which already contributed.

On the other hand stuffing what we want into `CONTRIBUTING.md` has a proven track record of not to work...

**Pull request merged**